### PR TITLE
Enhancement: Add ergebnis/composer-normalize as tool

### PR DESCRIPTION
**Pull request by localheinz**
---
name:  🎉 New Feature
about: I want to add a new feature
labels: enhancement
---

## A Pull Request should be associated with an Issue.

Related issue: #250

### Description

This PR adds `ergebnis/composer-normalize` as a tool

- [x] I have written test cases for the changes in this pull request
- [x] I have run `npm run format` before the commit.
- [x] I have run `npm run lint` before the commit.
- [x] I have run `npm run release` before the commit.
- [x] `npm test` returns with no unit test errors and all code covered.

### CODE REVIEW

File: __tests__/tools.test.ts [hunk 1]
```diff
@@ -447,7 +447,7 @@ describe('Tools tests', () => {
   });
   it('checking addTools on darwin', async () => {
     const script: string = await tools.addTools(
-      'blackfire, blackfire-player, flex, infection, phan, phpcs, phpcbf, phpcpd, phpmd, psalm, phinx, vapor-cli, phan:2.7.2, phive:1.2.3, cs2pr:1.2.3, composer-prefetcher:1.2.3, phpize, php-config, symfony, symfony:1.2.3, wp-cli',
+      'blackfire, blackfire-player, composer-normalize, flex, infection, phan, phpcs, phpcbf, phpcpd, phpmd, psalm, phinx, vapor-cli, phan:2.7.2, phive:1.2.3, cs2pr:1.2.3, composer-prefetcher:1.2.3, phpize, php-config, symfony, symfony:1.2.3, wp-cli',
```
Comment (localheinz) [inline __tests__/tools.test.ts:450]: The items in this string (and elsewhere) are not consistently ordered. I added `composer-normalize` in order.

**Comment by localheinz** [comment]
Thank you, @shivammathur!

**Comment by Slamdunk** [comment]
Hi, we have this check running from a month ago:

https://github.com/paratestphp/paratest/blob/5bed55731808bd48832d344311e27b0b11b4ecdd/.github/workflows/integrate.yaml#L49

And now I'm getting `/usr/local/bin/composer-normalize: Permission denied`:

https://github.com/paratestphp/paratest/pull/508/checks?check_run_id=992497289#step:8:5

Any guess?

**Comment by localheinz** [comment]
@Slamdunk 

The latest release workflow failed to sign and upload `composer-normalize.phar` - this is probably the root cause.

Taking a look!

**Comment by shivammathur** [comment]
@localheinz Please add the phar manually to the release till the issue is fixed.

**Comment by localheinz** [comment]
@shivammathur 

Done!

@Slamdunk 

Can you check again?

**Comment by Slamdunk** [comment]
Everything looks ok now :+1:

**Comment by localheinz** [comment]
Thank you, @Slamdunk!

**Pull request merged**